src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1565,6 +1598,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
 
         _renderer->TriggerSelection();
+        _UpdateSelectionMarkersHandlers(*this, winrt::make<implementation::UpdateSelectionMarkersEventArgs>(true));

```

Comment:
Almost seems like there should be a helper that calls both `TriggerSelection` and `_UpdateSelectionMarkersHandlers`, to make sure everywhere we call one, we call the other (or have an explicit reason to only be calling one)

Reply:
Actually, I just added it in and I found out that there were a few I was missing. Good call.